### PR TITLE
BugFix: matlab functions in packages are now handled correctly

### PR DESCRIPTION
**Pull request by pgagarinov**
Fixes issue 6

### CODE REVIEW

File: Doxyfile [hunk 1]
```diff
@@ -790,7 +790,8 @@ WARN_LOGFILE           =
 # spaces. See also FILE_PATTERNS and EXTENSION_MAPPING
 # Note: If this tag is empty the current directory is searched.
 
-INPUT                  = ./msource
+INPUT                  = C:\Users\Administrator\Documents\_Git\_MatrixBerry\test
```
Comment (simgunz) [inline Doxyfile]: Can you please revert this line to ./msource

File: Doxyfile [hunk 2]
```diff
@@ -790,7 +790,8 @@ WARN_LOGFILE           =
...
+#INPUT = C:\Users\Administrator\Documents\_Git\_MatrixBerry\mxberrybase3
```
Comment (simgunz) [inline Doxyfile]: and remove this one

**Comment by simgunz** [comment]
Thanks for the contribution.

File: Doxyfile (hunk 2 above)
Comment (pgagarinov) [reply Doxyfile]: done

**Pull request merged**